### ISSUE TITLE
Execute UTL_FILE.FCLOSE after executing UTL_FILE.FCLOSE, the application crashes.

### ISSUE THREAD

**Issue by naoki4444** [orafce/orafce#260]
If I use the UTL_FILE package feature and execute UTL_FILE.FCLOSE followed by UTL_FILE.FCLOSE_ALL, the application crashes.

The following is a sample program and execution results for reproduction.

* sample program

```
INSERT INTO UTL_FILE.UTL_FILE_DIR(dir) VALUES('/dev');

CREATE OR REPLACE  FUNCTION read_file(fname VARCHAR) RETURNS void AS $$
DECLARE
    v_file      UTL_FILE.FILE_TYPE;
    v_exists    BOOLEAN;
    v_length    NUMERIC;
    v_bsize     INTEGER;
    v_rbuff     VARCHAR(1024);
BEGIN
    PERFORM DBMS_OUTPUT.SERVEROUTPUT(TRUE);

    v_file := UTL_FILE.FOPEN('/dev', fname, 'w', 1024);
    PERFORM UTL_FILE.PUT_LINE (v_file, 'This is the third line', FALSE);
    v_file := UTL_FILE.FCLOSE(v_file);
    RETURN;
EXCEPTION
    WHEN OTHERS THEN
        PERFORM DBMS_OUTPUT.PUT_LINE('-- SQL Error --');
        PERFORM DBMS_OUTPUT.PUT_LINE('ERROR : ' || SQLERRM );
        PERFORM UTL_FILE.FCLOSE_ALL();
        RETURN;
END;
$$ 
LANGUAGE plpgsql;
```

* Execution result

```
postgres=# SELECT read_file('full');
-- SQL Error --
ERROR : UTL_FILE_WRITE_ERROR (10337)
server closed the connection unexpectedly
        This probably means the server terminated abnormally
        before or while processing the request. (14989)
```

UTL_FILE.FCLOSE results in UTL_FILE_WRITE_ERROR, and the EXCEPTION clause UTL_FILE.FCLOSE_ALL is executed, causing a crash.

**Comment by okbob** [comment]
25ec367a3ded6c2563f502fca6997e991d6e46e7 should to fix this issue

**Comment by naoki4444** [comment]
Thanks for the fix.

**Comment by okbob** [comment]
fix released in 4.9.4

**Comment by okbob** [comment]
thank you for report
